_data/devops-checklist.yml

```diff
@@ -0,0 +1,690 @@
+        <a href="https://gruntjs.com/" target="_blank">Grunt</a>,
+        <a href="https://gulpjs.com/" target="_blank">Gulp</a>,
+        <a href="https://github.com/broccolijs/broccoli" target="_blank">Broccoli</a>, or
+        <a href="https://webpack.js.org/" target="_blank">webpack</a>.
+
+    - title: Use a static content server
+      description: |
+        You should serve all your static content (CSS, JS, images, fonts) from a static content server so that your
+        dynamic web framework (e.g., from Rails, Node.js, or Django) can focus solely on processing dynamic requests.
+        The best static content host to use with AWS is <a href="https://aws.amazon.com/s3/" target="_blank">S3</a>.
+
+    - title: Use a CDN
+      description: |
+        Use <a href="https://aws.amazon.com/cloudfront/" target="_blank">CloudFront</a> as a
+        <a href="https://en.wikipedia.org/wiki/Content_delivery_network" target="_blank">Content Distribution Network (CDN)</a>
+        to cache and distribute your content across servers all over the world. This signicantly reduces latency for
+        users and is especially effective for static content.
+
+    - title: Configure caching
+      description: |
+        Think carefully about versioning, caching, and cache-busting for your static content. One option is to
+        put the version number of each release directly in the URL (e.g., <code>/static/v3/foo.js</code>), which
+        is easy to implement, but means 100% of your content is "cache buested" each release. Another option is "asset
+        fingerprinting," where the build system renames each static content file with a hash of that files contents
+        (e.g., <code>foo.js</code> becomes <code>908e25f4bf641868d8683022a5b62f54.js</code>), which is more complicated
+        to implement (note: many build systems have built-in support), but ensures that only content that has changed
+        is ever cache busted.
+
+- category: Data storage
+  items:
+    - title: Deploy relational databases
+      description: |
+        Use Amazon's <a href="https://aws.amazon.com/rds/" target="_blank">Relational Database Service (RDS)</a> to run
+        MySQL, PostgreSQL, Oracle, SQL Server, or MariaDB. Consider
+        <a href="https://aws.amazon.com/rds/aurora/" target="_blank">Amazon Aurora</a> as a highly scalable,
+        cloud-native, MySQL and PostgreSQL compatible database. Both RDS and Aurora support automatic failover, read
+        replicas, and automated backup.
+
+    - title: Deploy NoSQL databases
+      description: |
+        Use <a href="https://aws.amazon.com/elasticache/" target="_blank">Elasticache</a> if you want to use Redis or
+        Memcached for key-value storage. Use <a href="https://aws.amazon.com/dynamodb/" target="_blank">DynamoDB</a> if
+        you need a managed, eventually consistent document store. If you need other NoSQL databases, such as MongoDB
+        or Couchbase, you'll need to run it yoruself (see the
```

Comment:
run it ~yoruself~ yourself...

Reply:
Fixed! https://github.com/gruntwork-io/gruntwork-io.github.io/pull/45/commits/ac969266fa45512affe5434f70111b488f8c4eb8